web-admin/src/features/alerts/CreateAlert.svelte

```diff
@@ -16,27 +18,30 @@
   // This ensures Rill Developer doesn't try and fail to import the admin-client.
   let CreateAlertDialog;
   onMount(async () => {
-    CreateAlertDialog = (await import("./CreateAlertDialog.svelte")).default;
+    CreateAlertDialog = (
+      await import(
+        "@rilldata/web-common/features/alerts/CreateAlertDialog.svelte"
+      )
+    ).default;
   });
```

Comment:
Now that we're moving this entrypoint to a `web-admin`-specific component, we can remove this dynamic/conditional import.